Assets/MRTK/Core/Inspectors/PropertyDrawers/TypeReferencePropertyDrawer.cs

```diff
@@ -289,17 +289,22 @@ private static Type[] FindTypesByName(string typeName, SystemTypeAttribute filte
         private static void DisplayDropDown(Rect position, List<Type> types, Type selectedType, TypeGrouping grouping)
         {
             var menu = new GenericMenu();
-            menu.AddItem(new GUIContent("(None)"), selectedType == null, OnSelectedTypeName, null);
```

Comment:
Does this technically supercede https://github.com/microsoft/MixedRealityToolkit-Unity/pull/9125? Since, if it's no longer possible to _not_ select a gaze provider, we don't need to account for that case?

Reply:
I think so, though the Null checks and the Unit test are still valuable enough to still get in I think, and they don't introduce regressions to the current behavior. The GazeProvider becoming null is a symptom of the GazeProviderType getting unset, however, there could be other ways by which the GazeProvider suddenly becomes null.